platforms/nuttx/src/px4/nxp/s32k1xx/io_pins/io_timer.c

```diff
@@ -641,6 +639,14 @@ int io_timer_init_timer(unsigned timer)
 		if (handler) {
 			irq_attach(io_timers[timer].vectorno, handler, NULL);
 			up_enable_irq(io_timers[timer].vectorno);
+#if defined(CONFIG_ARCH_CHIP_S32K14X) /* Quick (and not very nice) workaround for the multiple interrupts per FTM on S32K14X */
+			irq_attach(io_timers[timer].vectorno + 1, handler, NULL);
```

Comment:
I'm still not really happy about this quick workaround. On S32K14X each FlexTimer has multiple interrupts (While S32K11X only has one per FTM). I just register and enable all of them, but that's probably not something you want to do if you're using the same FlexTimer for other purposes. We can just leave it until we need to change it, but I'm open to suggestions to improve this.

Reply:
The interrupts are only used for Capture. Which is not implemented. https://github.com/PX4/Firmware/blob/a4f7b1741024f57649f2e9f86381c949b4b661ec/platforms/nuttx/src/px4/nxp/s32k1xx/io_pins/io_timer.c#L151

In the io_timer_set_enable we do this, by timer or channel. But I would have to be clear on what the HW difference is.